web/templates/about-zh-CN.md

```diff
@@ -0,0 +1,54 @@
+## 批判性思维
+
+批判性思维是一种智力训练过程，即能够积极并富于技巧地概念化、应用、分析、综合和评估那些通过观测、实验、反思、推理或者交流而获得或产生的各种信息，从而来指引信念和行动。其典范形式，就是基于超越具体问题之上的通用智力价值标准：即简明、准确、精确、连贯、相关、坚实的证据、严密的推理，以及深度、广度和公平性。
+
+## 命题绘图
+
+<img style="width: 40%; margin-left: 4%;" align="right" src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/99/Whatley.png/800px-Whatley.png">
+
+
+命题绘图是制作“方框和箭头”推理图，尤其是复杂的争议和辩论。命题绘图提高了我们阐述，理解和交流论证的能力，从而促进批判性思维能力。
+你可以把命题绘图视为视觉层次的绘图。
+
+Arguman.org 的目标是让更多争议被用户发掘出来。
+
+## 我们是谁
+
+Arguman.org 是一个组织的开源项目, 如果你想贡献技术，智慧请点击如下链接：
```

Comment:
@kateliao0918 Good job. Note that in zh_CN we tend to use "开源" for "open source". Despite the differences between zh_CN and zh_TW, it will be helpful if we have joint discussions on core terms.  

@taiansu how is the traditional Chinese version going? :grinning: